doc/getting_started/setup_fpga.md

```diff
@@ -21,13 +21,37 @@ Refer to the design documentation for information what exactly is needed.
 
 * [Obtain an FPGA board]({{< relref "fpga_boards.md" >}})
 
-## Create an FPGA bitstream
+## Obtain an FPGA bitstream
+
+To run OpenTitan on an FPGA, you will need an FPGA bitstream.
+You can either download the latest bitstream for the ChipWhisperer CW310 board or build it yourself.
+
+### Download a Pre-built Bitstream
+
+If you are using the ChipWhisperer CW310 board with the Xilinx Kintex 7 XC7K410T FPGA, you can download the latest passing [pre-built bitstream](http://storage.googleapis.com/opentitan-bitstreams/master/latest/latest-bitstreams.tar.gz). 
+
+For example, to download the bitstream, run the following:
+
+```console
+$ cd $REPO_TOP
+$ mkdir -p build-bin/hw/top_earlgrey
+$ cd build-bin/hw/top_earlgrey
+$ curl https://storage.googleapis.com/opentitan-bitstreams/master/latest/latest-bitstreams.tar.gz -o latest-bitstreams.tar.gz
+$ tar -xvf latest-bitstreams.tar.gz
+```
+
+By default, the bitstream contains a version of the boot ROM used for testing (pulled from `sw/device/lib/testing/test_rom`).
+The included `rom.mmi` file is a version of the production mask ROM that can be spliced into the bitstream.
```

Comment:
```suggestion
The included `rom.mmi` file is a version of the boot ROM used in production (called the _mask ROM_; pulled from `sw/device/silicon_creator/mask_rom`). This can be spliced into the bitstream to overwrite the testing boot ROM using a script in our repo.
```